### PR TITLE
data: SumuSiriwardana

### PR DESCRIPTION
**Pull request by sumusiriwardana**

### CODE REVIEW

File: data/sumusiriwardana.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Project Manager | Web Developer | Technical Writer",
+  "avatar": "https://github.com/sumusiriwardana.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sumusiriwardana",
+      "icon": "FaTwitter"
+    }
```
Comment (Pradumnasaraf) [inline data/sumusiriwardana.json:12]:
```suggestion
    },
```

File: data/sumusiriwardana.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sumusiriwardana",
+      "icon": "FaGithub"
+    }
```
Comment (Pradumnasaraf) [inline data/sumusiriwardana.json:17]:
```suggestion
    },
```

File: data/sumusiriwardana.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow my blog on Hashnode",
+      "url": "https://sumudusiriwardana.hashnode.dev/",
+      "icon": "SiHashnode"
+    }
```
Comment (Pradumnasaraf) [inline data/sumusiriwardana.json:22]:
```suggestion
    },
```

File: data/sumusiriwardana.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow my blog on Dev.to",
+      "url": "https://dev.to/sumusiriwardana",
+      "icon": "FaDev"
+    }
```
Comment (Pradumnasaraf) [inline data/sumusiriwardana.json:27]:
```suggestion
    },
```

File: data/sumusiriwardana.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow my blog on Medium",
+      "url": "https://medium.com/@sumudusiriwardana",
+      "icon": "FaMedium"
+    }
```
Comment (Pradumnasaraf) [inline data/sumusiriwardana.json:32]:
```suggestion
    },
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

There are some missing `,` in the file. I have suggested changes, please update them.

If you need any help, feel free to ask.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by eddiejaoude** [comment]
@sumusiriwardana is here 🎉

**Pull request merged**